code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -64,6 +64,13 @@
 	desc = "It's Officer Pingsky! Delegated to satellite guard duty for harbouring anti-human sentiment."
 	radio_channel = RADIO_CHANNEL_AI_PRIVATE
 
+/mob/living/simple_animal/bot/secbot/armsky //NSV13
+	name = "Sergeant-At-Armsky"
+	health = 45
+	idcheck = TRUE
+	weaponscheck = TRUE
+	arrest_type = TRUE
+
```

Comment:
I'd rather you define this in an NSV file, as it isn't using any of the code above or below it. Alternatively porting it upstream is also an option.

Reply:
Went with the former. Is there anything else I should do?

Reply:
Nope, that's it!